Gemfile

```diff
@@ -6,6 +6,7 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
+  gem 'activesupport', '~> 7.2'
```

Comment:
The most recent version of this gem is 8.0. Ruby version needs to be increased to 3.2.

Reply:
I think this should be moved to the [gemspec](https://github.com/passageidentity/passage-ruby/blob/cad1b0591537b93cab13babebab6c21d2179cd09/passageidentity.gemspec#L26) instead since this section is only for dev dependencies

Reply:
100% good catch.